### DISCUSSION TITLE
classes/class_physicstestmotionparameters3d

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#171] [User-contributed notes]
# classes/class_physicstestmotionparameters3d

Inherits: RefCounted< Object Provides parameters for PhysicsServer3D.body_test_motion. Description: By changing various properties of this object, such as the motion, you can configure the paramete...

https://docs.godotengine.org/en/stable/classes/class_physicstestmotionparameters3d.html

**Comment by stupiddude1234568909** [comment]
i don't know how to sugest this a contribution, but i don't think the documentation is clear about the fact that you continually have to update the from parameter with global_transform.

**Comment by macbodziak** [comment]
Also worth noting, that motion is in world space, not local to the from transform.
